src/components/TextInput/BaseTextInput.js

```diff
@@ -316,6 +317,12 @@ class BaseTextInput extends Component {
     }
 }
 
+// Disable font scaling for react native Text/TextInput components because we are handling it manually
+RNText.defaultProps = RNText.defaultProps || {};
+RNText.defaultProps.allowFontScaling = false;
+TextInput.defaultProps = TextInput.defaultProps || {};
+TextInput.defaultProps.allowFontScaling = false;
```

Comment:
Please move the logic to the component level and not here.

Reply:
Done.